### PR TITLE
[16.0][MIG] maintenance_timesheet_time_control: Migration to 16.0

### PR DESCRIPTION
**Pull request by victoralmau**
Migration to 16.0

Please @CarlosRoca13 and @carolinafernandez-tecnativa can you review it?

@Tecnativa TT48099

### CODE REVIEW

**Review by CarlosRoca13 — APPROVED** [review]

File: maintenance_timesheet_time_control/tests/test_maintenance_timesheet_time_control.py [hunk 1]
```diff
@@ -1,44 +1,41 @@
...
-            groups="%s,%s" % (maintenance_group, timesheet_group),
-            context=ctx,
-        )
-        cls.employee = cls.env["hr.employee"].create(
-            {"name": "Test employee", "user_id": cls.user.id}
+            groups="%s,%s"
+            % (
+                "maintenance.group_equipment_manager",
+                "hr_timesheet.group_hr_timesheet_user",
+            ),
```
Comment (pedrobaeza) [inline maintenance_timesheet_time_control/tests/test_maintenance_timesheet_time_control.py]: No need of complicating expressions:

```suggestion
            groups="maintenance.group_equipment_manager,"
            "hr_timesheet.group_hr_timesheet_user",

**Review by pedrobaeza — COMMENTED** [review]
Please add in the CONFIGURE.rst that the maintenance request must have a project defined for seeing the start/stop buttons.

**Comment by victoralmau** [comment]
Changes done.

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**